docs/spelling_wordlist

```diff
@@ -423,6 +423,7 @@ recomputation
 recursed
 redeclare
 redirections
+redis
```

Comment:
I think you can instead add Redis with a capital R only, to only permit it to be written with a capital letter. This is how https://redis.io/ writes it.

Reply:
I checked and it doesn't matter. `redis` is still valid, even with `Redis` in `spelling_wordlist`, but we can change it.

Reply:
No worries then.